e2e-tests/e2e-tests.ts

```diff
@@ -28,17 +28,25 @@ function delay (timeout) {
 }
 
 describe('A non-matching journey', () => {
-  it('should succeed', async () => {
-    await journey().then(async function () {
+  it('without eIDAS should succeed', async () => {
```

Comment:
Might read more clearly as "should succeed without eIDAS".

Reply:
The full sentence is `A non-matching journey without eIDAS should succeed`. It read better to me `... a journey with(out) eIDAS should succeed` ... Happy to change though

Reply:
It doesn't read so well if you include the Mocha keywords; I think it's supposed to.  The underlying issue is that we're trying to modify the noun phrase (a non-matching journey) with an adjectival phrase (without eIDAS) in individual tests; probably what Mocha would prefer would be for us to have two separate noun phrases:

`describe('A non-matching journey without eIDAS')`

and

`describe('A non-matching journey with eIDAS')`

I don't mind if it gets left as it is though.

Reply:
Yeah, good point. I was basing the readability on the console output:

<img width="297" alt="image" src="https://user-images.githubusercontent.com/30629434/53171498-c8d42600-35da-11e9-9045-b76f1634b454.png">
Let me update it ;)